schedule/containers/helm.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: helm
+description: |
+  helm test
+schedule:
```

Comment:
after a chat, agreed to use `main_containers.pm` instead of yaml schedule.

Reply:
done